packages/migrate/src/__tests__/MigrateDev.test.ts

```diff
@@ -1126,17 +1120,22 @@ describe('postgresql', () => {
 })
 
 describe('mysql', () => {
+  const connectionString = (
+    process.env.TEST_MYSQL_URI_MIGRATE || 'mysql://root:root@localhost:3306/tests-migrate'
+  ).replace('tests-migrate', 'tests-migrate-dev')
+
+  // Update env var because it's the one that is used in the schemas tested
+  process.env.TEST_MYSQL_URI_MIGRATE = connectionString
```

Comment:
I prefer this format https://github.com/prisma/prisma/pull/11524/files#diff-c266a026f47d4de125b7139076b575b33fb6dcb0207110c4a9a0f78304097b41R1335

Reply:
Yeah, I have no super strong opinion on this, but I personally think the default hardcoded connection strings are not super useful and I'd personally prefer the test to fail if the env var is not set. That the absence of the default value would make the second format (which is shorter and more readable) possible here is a nice bonus.

Reply:
Although that format is possible even with the default value, like this:

```ts
process.env.TEST_MYSQL_URI_MIGRATE = process.env.TEST_MYSQL_URI_MIGRATE?.replace(
  'tests-migrate',
  'tests-migrate-dev'
) ?? 'mysql://root:root@localhost:3306/tests-migrate-dev'
```

Reply:
Yes I think we can get rid of all the defaults at some point in a separate PR @aqrln, it's hard to maintain them (might not be in sync) and a clear single point of config is better